Userland/Libraries/LibTest/TestSuite.cpp

```diff
@@ -262,4 +281,4 @@ int TestSuite::run(Vector<NonnullRefPtr<TestCase>> const& tests)
     return (int)(test_count - test_passed_count + benchmark_count - benchmark_passed_count);
 }
 
-}
+} // namespace Test
```

Comment:
This should be added when the namespace was first added (i.e. amended into an earlier commit).

Reply:
I'm unsure, maybe in one of my previous rebases this comment got moved to some further commit, but right now it's in the first commit of the PR. I can't go earlier than that